src/config.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Parser;
+
+/// Define a struct to hold all command-line arguments
+#[derive(Debug, Parser)]
+#[command(name = "Nomad External DNS Tool For Hetzner", version = "0.1.0", about = "Automatically updates DNS records", long_about = None)]
+pub struct Config {
+    /// Sets the Hetzner DNS API token
+    #[arg(long)]
+    pub hetzner_dns_token: String,
```

Comment:
I suggest allowing at least the token to be provided via environment variable for slightly better security. See [here](https://github.com/svenstaro/miniserve/blob/2713890568adb679ab297a874a37a14307425cec/src/args.rs#L22) for an example.